plotnine/geoms/geom_map.py

```diff
@@ -136,7 +136,10 @@ def draw_panel(self, data, panel_params, coord, ax, **params):
             segments = []
             for g in data['geometry']:
                 if g.geom_type == 'LineString':
-                    segments.append(g.coords)
+                    if g.has_z:
```

Comment:
Add a warning of type `PlotnineWarning` that the `z` coordinate is being ignored.

Reply:
Also add test in [test_geom_map.py](https://github.com/has2k1/plotnine/blob/15b9759047a46523d9c5a525f6f60dac24b6b72a/plotnine/tests/test_geom_map.py).